salt/modules/zabbix.py

```diff
@@ -1489,6 +1489,290 @@ def hostinterface_update(interfaceid, **connection_args):
         return ret
 
 
+def usermacro_get(macro=None, hostids=None, templateids=None, hostmacroids=None,
```

Comment:
I think it would be much better to separate these out so that the words aren't run together. So instead of `hostids` it would be `host_ids`.